src/Header.css

```diff
@@ -1,10 +1,32 @@
-.Header {
+header {
   position: relative;
+  /*background: rgba(74, 1, 96, .9);*/
+  /*color: #ffffff;*/
+  color: rgb(62, 179, 223);
```

Comment:
Looks good! Only thing I might say is we could probably clean out the commented out code here. And I'm pretty sure since we added position: relative to the logo now we should only need a z-index on the header to pull it above the pseudoelement, and not on the header elements themselves.